toolkit/scripts/srpm_pack.mk

```diff
@@ -17,13 +17,16 @@ local_specs = $(shell find $(SPECS_DIR)/ -type f -name '*.spec')
 local_spec_dirs = $(foreach spec,$(local_specs),$(dir $(spec)))
 local_sources = $(shell find $(SPECS_DIR)/ -name '*')
 
+toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
```

Comment:
>$(toolchain_build_dir) [](start = 22, length = 22)

I think `toolchain_build_dir` is only defined in `toolchain.mk`, so it might not be set here.

Reply:
Unfortunately Makefiles don't have a very robust include system, its pretty much just copy/pasting the files into the main Makefile where the their respective include line is. (ie, so long as `toolchain.mk` appears before `srpm_pack.mk` this is fine). This isn't the first time this has happened.

If stuff starts being used in more than one sub-script though I've tended towards moving the variables into the main Makefile. Having a separate variables file might be interesting, but probably not worth the churn right now.